package.json

```diff
@@ -170,10 +170,10 @@
     "@types/babel__core": "^7",
     "@types/jest": "^29.5.1",
     "@types/mysql": "^2",
-    "@types/node": "^18.18.2",
+    "@types/node": "18.16.9",
```

Comment:
Why the downgrade?

Reply:
Ah, must have happened as part of the nx migration. I've reverted the change.